src/pages/logos.tsx

```diff
@@ -0,0 +1,177 @@
+  size: string;
+}) {
+  const [modalState, setModal] = useState(false);
+  const modalRef = useRef<HTMLDivElement>(null);
+
+  const openModal = () => {
+    setModal(true);
+  };
+
+  const closeModal = () => {
+    setModal(false);
+  };
+
+  const handleOverlayClick = (event: React.MouseEvent) => {
+    if (modalRef.current === event.target) {
+      closeModal();
+    }
+  };
+
+  return (
+    <div>
+      <img
+        src={addImageDetails(pic)}
+        alt={companyName}
+        onClick={openModal}
+        className={`${size} cursor-pointer`}
+      />
+
+      {modalState && (
+        <div
+          className="fixed inset-0 flex items-center justify-center bg-black bg-opacity-60"
+          ref={modalRef}
+          onClick={handleOverlayClick}
+        >
+          <div
+            className={`bg-black bg-opacity-0 w-[300px] sm:w-[500px] pt-10 pb-5 flex flex-col rounded-3xl`}
+          >
+            <div className="relative py-[0px] justify-center flex flex-row">
+              <img src={addImageDetails(pic)} alt={companyName} />
+              {/* 
+              <button
+                className="absolute top-5 right-3 w-[50px] h-[50px] flex items-center justify-center"
+                onClick={closeModal}
+              >
+                <div className="absolute h-[50px] w-[5px] bg-black rounded-md rotate-45"></div>
+                <div className="absolute h-[50px] w-[5px] bg-black rounded-md -rotate-45"></div>
+              </button>
+              */}
+            </div>
+            <div className="px-5 mt-5">
+              <h2 className="text-center text-3xl text-cerise">
+                {companyName}
+              </h2>
+              <p className="text-white text-center mt-5">{description}</p>
+            </div>
+          </div>
+        </div>
+      )}
+    </div>
+  );
+}
+
+function chunkArray<T>(array: T[], size: number): T[][] {
+  const chunkedArr: T[][] = [];
+  let index = 0;
+  while (index < array.length) {
+    chunkedArr.push(array.slice(index, size + index));
+    index += size;
+  }
+  return chunkedArr;
+}
+function RenderLogos(packageList: any[], rowSize: number, logoSize: string) {
+  return (
+    <>
+      {chunkArray(packageList, rowSize).map((chunk, rowIndex) => (
+        <div
+          key={rowIndex}
+          className="flex flex-row justify-center items-center mb-4 gap-5 mt-[100px]"
+        >
+          {chunk.map((exhibitor, idx) => (
+            <div key={idx}>
+              <Logo
+                pic={exhibitor.logoColor} // exhibitor.logoColor
+                companyName={exhibitor.name}
+                description={exhibitor.description}
+                size={logoSize}
+              />
+            </div>
+          ))}
+        </div>
+      ))}
+    </>
+  );
+}
+type LogosProps = {
+  exhibitorData: {
+    id?: string;
+    name: string;
+    logoWhite?: string | null;
+    logoColor?: string | null;
+    description?: string;
+    package?: string;
```

Comment:
Let's use the `Package` type here to make it super clear.

Reply:
Fixed